app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def authorize_admin!
+    if !user_signed_in? && !current_user.admin?
+      # raise ActionController::RoutingError.new("Not Found")
```

Comment:
I'd advise deleting comments. Ruby is an expressive enough language we shouldn't need to be leaving comments in our code. And if you are worried about forgetting that code write it down somewhere, or write down how you got it, but it'll be in GitHub history! Once it's in Git code is forever! :)

But yea, remove comments